lib/schema/facter.yaml

```diff
@@ -184,6 +184,15 @@ chassistype:
     caveats: |
         Linux: kernel 2.6+ is required due to the reliance on sysfs.
 
+cloud:
+    type: map
+    description: Information about the cloud instance of the node.
```

Comment:
Can we make it clearer that this only supports Azure at the moment? This becomes the documentation for the fact.